decision_service/examples/rl_sim_cpp/person.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <string>
+#include <unordered_map>
+class person {
+  public:
+    using topic_prob = std::unordered_map<std::string, float>;
+    person(std::string, std::string,
```

Comment:
It may be helpful to include the parameter names for the constructor in the header for people reading the header for documentation

Reply:
Done.